src/client/view/mostUsedWords.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { SlidingSearchBar } from './searchBar';
+
+const WordInfo = ({ wordInfo }) => (
+  <tr>
+    <td className='right aligned column'>{wordInfo.count}</td>
+    <td className='left aligned column'>{wordInfo.word}</td>
+    <td className='right aligned column'>
+      <div className='ui checkbox'>
+        <input type='checkbox'  defaultChecked='true'/>
+      </div>
+    </td>
+  </tr>
+);
+
+const Words = React.createClass({
+  componentDidMount() {
+    $('.ui.checkbox').checkbox();
+  },
+
+  componentDidUpdate() {
+    $('.ui.checkbox').checkbox();
+  },
+
+  render() {
+    return (
+      <div style={{ 'height': '300px', 'overflow-y': 'scroll' }}>
+        <table className='ui very basic celled table'>
+          <tbody>
+          { this.props.words.map((wordInfo) => <WordInfo wordInfo={wordInfo}></WordInfo>) }
+          </tbody>
+        </table>
+      </div>
+    );
+  },
+});
+
+const ToggleAllWords = () => (
+  <button className='ui button'>Hide all</button>
+);
+
+export const MostUsedWords = React.createClass({
+  getInitialState() {
+    return { 'search': '' };
+  },
+
+  filterWords() {
+    return this.props.words
+      .filter((word) => word.word.toLowerCase().includes(this.state.search.toLowerCase()))
+      .slice(0, 100);
+  },
+
+  render() {
+    return (
+      <div>
+        <h3>Most frequent words</h3>
+        <div>
+          <div className='ui two column grid'>
+            <div className='column'>
+              <SlidingSearchBar parentComp={this}/>

```

Comment:
better to explicitly pass down prop... either when it changes to prop then prop otherwise this.search to make it explicit